grammar.js

```diff
@@ -2092,26 +2101,29 @@ module.exports = grammar({
       ),
       optional($.where),
       optional($.group_by),
+      optional($.window_clause),
       optional($.order_by),
       optional($.limit),
     ),
 
-    relation: $ => seq(
-      choice(
-        $.subquery,
-        $.invocation,
-        $.table_reference,
-        seq(
-          '(',
-          $.values,
-          ')',
+    relation: $ => prec.right(
+      seq(
+        choice(
+          $.subquery,
+          $.invocation,
+          // TODO already has an optional alias
```

Comment:
whats the TODO here?

Reply:
I think this one's leftover from when `table_reference` included an alias as well as there being the alias here

```suggestion
```